### PR TITLE
docs: PL/Python, PL/R updates for installing modules and using gpssh.

### PR DESCRIPTION
**Pull request by mkiyama**
-remove compiling Python numpy package, add using pip to install numpy.
-mention gpssh -s option to source greenplum_path.sh

PR for 5X_STABLE
Will be ported to MAIN

Link to HTML output on GPDB 
http://docs-gpdb-review-staging.cfapps.io/550/ref_guide/extensions/pl_python.html
http://docs-gpdb-review-staging.cfapps.io/550/ref_guide/extensions/pl_r.html

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/extensions/pl_python.xml [hunk 1]
```diff
@@ -326,229 +326,105 @@ $$;</codeblock>
...
       <codeblock>/<varname>path_to_greenplum-db</varname>/ext/python/bin/python</codeblock>
+      <p>When running shell commands on remote hosts with <codeph>gpssh</codeph>, you can specify
+        the <codeph>-s</codeph> option. When the option is specified, <codeph>gpssh</codeph> sources
+        the <codeph>greenplum_path.sh</codeph> file before running commands on the remote hosts. For
+        example, this command should display the Python installed with Greenplum Database on each
+        host.<codeblock>gpssh -f gpdb_hosts which python</codeblock></p>
+      <p>If it does not, you can add the <codeph>-s</codeph> to source
+          <codeph>greenplum_path.sh</codeph> on the remote hosts before running the command.
+        <codeblock>gpssh -s -f gpdb_hosts which python</codeblock></p>
+      <p>To display the list to currently installed Python modules, run this command.</p>
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/extensions/pl_python.xml]: change from "list to" to "list of"

File: gpdb-doc/dita/ref_guide/extensions/pl_python.xml [hunk 2]
```diff
@@ -326,229 +326,105 @@ $$;</codeblock>
...
->>> exit</codeblock></li>
-        </ol>
-        <p>The <codeph>setuptools</codeph> package installs the <codeph>easy_install</codeph>
-          utility that lets you install Python packages from the Python Package Index repository.
-          For example, this command installs Python <codeph>PIP</codeph> utility from the Python
-          Package Index site.<codeblock>easy_install pip</codeblock></p>
-        <p>You can use the <codeph>gpssh</codeph> utility to run the <codeph>easy-install</codeph>
-          command on all the Greenplum Database segment hosts.</p>
+        <p>The Python utility <codeph>pip</codeph> installs Python packages that contain Python
+          modules and other and other resource files from versioned archive files. </p>
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/extensions/pl_python.xml]: change "and other and other" to "and other"

File: gpdb-doc/dita/ref_guide/extensions/pl_python.xml [hunk 3]
```diff
@@ -326,229 +326,105 @@ $$;</codeblock>
...
+        <p>Run this command to install <codeph>pip</codeph>.</p>
+        <codeblock>python -m ensurepip --default-pip</codeblock>
+        <p>The command runs the <codeph>ensurepip</codeph> module to bootstrap (install and
+          configure) the <codeph>pip</codeph> utility from the local Python installation.</p>
+        <p>You can run this command ensure the <codeph>pip</codeph>, <codeph>setuptools</codeph> and
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/extensions/pl_python.xml]: change "run this command ensure" to "run this command to ensure"

File: gpdb-doc/dita/ref_guide/extensions/pl_python.xml [hunk 4]
```diff
@@ -326,229 +326,105 @@ $$;</codeblock>
...
+[centos6-sdw1] Collecting pip
+[centos6-sdw1] Installing collected packages: setuptools, pip
+[centos6-sdw1] Successfully installed pip-8.1.1 setuptools-20.10.1
+=> exit
+$</codeblock></p>
+        <p>The <codeph>=></codeph> is the inactive prompt for <codeph>gpssh</codeph>. The utility
+          displays the output from each host. The <codeph>exit</codeph> command exits from
+            <codeph>gpssh</codeph> interactive mode.</p>
+        <p>This <codeph>gpssh</codeph> command runs a single command on all hosts listed in the file
+            <codeph>gpdg_hosts</codeph>.<codeblock>gpssh -s -f gpdb_hosts python -m pip install --upgrade pip setuptools wheel</codeblock></p>
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/extensions/pl_python.xml]: change "gpdg_hosts" to "gpdb_hosts"

File: gpdb-doc/dita/ref_guide/extensions/pl_python.xml [hunk 5]
```diff
@@ -326,229 +326,105 @@ $$;</codeblock>
...
+        <p>The utility displays the output from each host.</p>
+        <p>For more information about installing Python packages see <xref
+            href="https://packaging.python.org/tutorials/installing-packages/" format="html"
+            scope="external">https://packaging.python.org/tutorials/installing-packages/</xref></p>
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/extensions/pl_python.xml]: add a period before <p>

File: gpdb-doc/dita/ref_guide/extensions/pl_python.xml [hunk 6]
```diff
@@ -693,23 +561,16 @@ $$ language plpythonu;</codeblock>
...
-              downloads the nltk package tar
-                file.<codeblock>wget http://pypi.python.org/packages/source/n/nltk/nltk-2.0.2.tar.gz#md5=6e714ff74c3398e88be084748df4e657</codeblock><note>The
-                Python package Distribute <xref href="https://pypi.python.org/pypi/distribute"
-                  format="html" scope="external">https://pypi.python.org/pypi/distribute</xref> is
-                required for nltk. The Distribute module should be installed the ntlk package. This
-                  <codeph>wget</codeph> command downloads the Distribute package tar
-                file.<codeblock>wget http://pypi.python.org/packages/source/d/distribute/distribute-0.6.21.tar.gz</codeblock></note></li>
+                scope="external">http://www.nltk.org/</xref>. For information about installing the
+              toolkit see <xref href="http://www.nltk.org/install.html" format="html"
+                scope="external">http://www.nltk.org/install.html</xref></li>
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/extensions/pl_python.xml]: add a period before </li>

File: gpdb-doc/dita/ref_guide/extensions/pl_python.xml [hunk 7]
```diff
@@ -326,229 +326,105 @@ $$;</codeblock>
...
+[centos6-sdw1] Collecting setuptools
+[centos6-sdw1] Collecting pip
+[centos6-sdw1] Installing collected packages: setuptools, pip
+[centos6-sdw1] Successfully installed pip-8.1.1 setuptools-20.10.1
+=> exit
+$</codeblock></p>
+        <p>The <codeph>=></codeph> is the inactive prompt for <codeph>gpssh</codeph>. The utility
+          displays the output from each host. The <codeph>exit</codeph> command exits from
+            <codeph>gpssh</codeph> interactive mode.</p>
+        <p>This <codeph>gpssh</codeph> command runs a single command on all hosts listed in the file
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/extensions/pl_python.xml]: Missing period or colon in this sentence.

File: gpdb-doc/dita/ref_guide/extensions/pl_r.xml [hunk 8]
```diff
@@ -253,10 +250,11 @@ CREATE TABLE wj_model_results_roi AS SELECT *
           <li id="py219006">From the command line, use the <codeph>wget</codeph> utility to download
             the <codeph>tar.gz</codeph> files for the arm package to the Greenplum Database master
             host:<codeblock>wget http://cran.r-project.org/src/contrib/Archive/arm/arm_1.5-03.tar.gz</codeblock><codeblock>wget http://cran.r-project.org/src/contrib/Archive/Matrix/Matrix_0.9996875-1.tar.gz</codeblock></li>
-          <li id="py219012">Use the <codeph>gpscp</codeph> utility and the hosts_all file to copy
-            the <codeph>tar.gz</codeph> files to the same directory on all nodes of the Greenplum
-            cluster. The <codeph>hosts_all</codeph> file contains a list of all the Greenplum
-            Database segment hosts. You might require root access to do
+          <li id="py219012">Use the <codeph>gpscp</codeph> utility and the
+              <codeph>hosts_all</codeph> file to copy the <codeph>tar.gz</codeph> files to the same
+            directory on all nodes of the Greenplum cluster. The <codeph>hosts_all</codeph> file
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/extensions/pl_r.xml]: Greenplum -> Greenplum Database

**Pull request merged**